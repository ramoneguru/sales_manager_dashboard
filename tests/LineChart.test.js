import React from 'react';
import { shallow, render, mount } from 'enzyme';
import LineChart from '../src/js/components/charts/LineChart';


describe('LineChart component ', function(){
  "use strict";

  it('should render correctly', () => {
    let activityEfficiency = {};
    let salesReps = {};
    
    const wrapper = shallow(
      <LineChart activityEfficiency={activityEfficiency} salesReps={salesReps}></LineChart>
    );

    expect(wrapper).toMatchSnapshot();
  });

});
