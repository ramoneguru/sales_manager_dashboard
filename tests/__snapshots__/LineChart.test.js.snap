// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LineChart component  should render correctly 1`] = `
<div>
  <figure
    className="chart-container chart-container--linechart"
  >
    <div
      className="loading-indicator animated-background end"
    >
      <div
        className="background-masker"
      />
      <div
        className="background-masker"
      />
      <div
        className="background-masker"
      />
      <div
        className="background-masker"
      />
    </div>
  </figure>
  <ul
    className="chart-key chart-key--linechart"
  >
    <li>
      <i
        className="call"
      />
       
      Call : Connect
    </li>
    <li>
      <i
        className="connect"
      />
       
      Connect : Meeting Schd.
    </li>
    <li>
      <i
        className="meeting schd."
      />
       
      Meeting Schd. : Opp Sourced
    </li>
  </ul>
</div>
`;
